test/distributed/_sharded_tensor/test_sharded_tensor.py

```diff
@@ -122,6 +124,52 @@ def wrapper(self):
         self.destroy_comms()
     return wrapper
 
+class TestShardedTensorMetadata(TestCase):
+    def test_serde(self):
```

Comment:
probably rename it to 

test_serialize_and_deserialize 

for better readability ?